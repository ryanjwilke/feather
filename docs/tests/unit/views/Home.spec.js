import { shallowMount } from '@vue/test-utils'
import Home from '@/views/home/home.vue'

describe('Home', () => {
  const wrapper = shallowMount(Home)

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
