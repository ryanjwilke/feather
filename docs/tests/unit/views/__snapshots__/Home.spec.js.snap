// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  class="home"
>
  <h1>
    Feather
  </h1>
   
  <h2>
    An incredibly lightweight, atomic CSS framework that's based entirely in Sass.
  </h2>
</div>
`;
